cvFit/src/app/app.module.ts

```diff
@@ -19,35 +19,62 @@ import { ExerciseListComponent } from './exercises/exercise-list.component';
 import { ExerciseDetailComponent } from './exercises/exercise-detail.component';
 import { WelcomeComponent } from './home/welcome.component';
 import { ContactUsComponent } from './form/contactUs.component';
-import { LoginComponent } from './form/login.component';
-import { ProfileComponent } from './form/profile.component';
 import { BlogComponent } from './blog/blog.component';
 import { WorkoutListComponent } from './exercises/workoutList/workoutList.component';
 import { PostComponent } from './blog/post/post.component';
 import { DiscussionComponent } from './blog/discussion/discussion.component';
-
-import { AuthService } from './form/auth.service';
 import { CalculatorComponent } from './macroCalculator/calculator.component';
 import { MacroTableComponent } from './macroCalculator/macro-table.component';
 import { CalcService } from './macroCalculator/calculator.service';
 
+import { MsalInterceptor, MsalModule, MsalRedirectComponent, MsalGuard } from '@azure/msal-angular';
+import { InteractionType, PublicClientApplication } from '@azure/msal-browser';
+import { AppRoutingModule } from './app-routing.module';
+import { AZUREprofileComponent } from './azureprofile/azureprofile.component';
+import { HomeComponent } from './home/home.component';
+
+const isIE = window.navigator.userAgent.indexOf('MSIE ') > -1 || window.navigator.userAgent.indexOf('Trident/') > -1;
+
 @NgModule({
   declarations: [
     AppComponent,
     ExerciseListComponent,
     ExerciseDetailComponent,
     ContactUsComponent,
     WelcomeComponent,
-    LoginComponent,
-    ProfileComponent,
     BlogComponent,
     WorkoutListComponent,
     PostComponent,
     DiscussionComponent,
     CalculatorComponent,
-    MacroTableComponent
+    MacroTableComponent,
+    AZUREprofileComponent,
+    HomeComponent
   ],
   imports: [
+    AppRoutingModule,
+    MsalModule.forRoot( new PublicClientApplication({
+      auth: {
+        clientId: '6063621e-ae81-4c3e-b0ee-6c9486c01725', // Application (client) ID from the app registration
+        //authority: 'Enter_the_Cloud_Instance_Id_Here/55e374bf-374e-49de-a716-836ce6f714d1', // The Azure cloud instance and the app's sign-in audience (tenant ID, common, organizations, or consumers)
+        authority: 'https://login.microsoftonline.com/55e374bf-374e-49de-a716-836ce6f714d1',
+        redirectUri: 'http://localhost:4200'// This is your redirect URI
```

Comment:
Redirect can just be `'/'`. You can also provide `postLogoutRedirectUri: '/'` so you don't have to set it in the logout request. putting localhost in the code just means it's not going to work if you deploy this app anywhere other than local.